source/includes/harvest/_demographic_data.md

```diff
@@ -21,8 +44,10 @@ Demographic questions and answers submitted during the application process. For
 | Attribute | Description |
 |-----------|-------------|
 | id | The demographic question's unique identifier |
-| translations.language | Only `en` (English) is supported at this time.
-| translations.name | The question text
+| question_set_id | The demographic question set that this belongs to
+| name | The question text
+| translations.language | Translations have been deprecated.  Only `en` (English) is supported at this time.
+| translations.name | Translations have been deprecated. This value will be the same as `name` above.
```

Comment:
Update to "Translations have been deprecated but are kept for backwards compatibility."